op-node/rollup/superchain.go

```diff
@@ -19,6 +19,7 @@ const (
 	opGoerli    = 420
 	opSepolia   = 11155420
 	baseGoerli  = 84531
+	baseSepolia = 84532
```

Comment:
The other unexported constants here are for a few chain-specific exceptions, for configuration edge-cases that are not supported in the superchain-registry repository. I believe we can omit any unused constants here, and just register the chains in the registry.

Reply:
happy to remove them and the other unused constants if preferred